includes/frontend.php

```diff
@@ -540,6 +540,13 @@ public function register_styles() {
 			ELEMENTOR_VERSION
 		);
 
+		wp_register_style(
+			'e-swiper',
+			$this->get_css_assets_url( 'e-swiper', 'assets/css/conditionals/' ),
+			[],
```

Comment:
Same here? should we add `swiper` dependency for `e-swiper`?

Reply:
Implemented